startups/lib/main_0.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:english_words/english_words.dart';
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Namer App',
+      theme: ThemeData(
+        useMaterial3: true,
+        colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepOrange),
+      ),
+      home: MyHomePage(),
+    );
+  }
+}
+
+class MyHomePage extends StatefulWidget {
+  @override
+  State<MyHomePage> createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  var current = WordPair.random();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Column(
```

Comment:
Starting with a column lets students play around with layouts early on, and also shows how to list children correctly.